op-devstack/sysgo/system.go

```diff
@@ -109,7 +109,7 @@ func NewDefaultSingleChainInteropSystemIDs(l1ID, l2AID eth.ChainID) DefaultSingl
 		L2AEL:         stack.NewL2ELNodeID("sequencer", l2AID),
 		L2ABatcher:    stack.NewL2BatcherID("main", l2AID),
 		L2AProposer:   stack.NewL2ProposerID("main", l2AID),
-		L2ChallengerA: "chainA",
+		L2ChallengerA: stack.NewL2ChallengerID("main", l2AID),
```

Comment:
not sure why the others use `main` as its key, but it should be:
```suggestion
		L2ChallengerA: stack.NewL2ChallengerID("challenger", l2AID),
```

so we can easily disambiguate the challenger from other nodes on the same chain.

Reply:
It already includes a "kind" parameter, so it prints out like: "L2Challenger-main-901".  Though I don't think "main" is very helpful 🤷‍♀️ Not sure what would really make sense here.  Maybe "chainA"?

Reply:
i missed the "kind" parameter. `main` is fine in that case.